### PR TITLE
Rules page

### PR DESCRIPTION
**Pull request by saicharan-gouru**
-Added rules data.
-Added rules image.
-Created rules page.
quiz rules

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ function App() {
       <Routes>
         <Route path="/" element={<Homepage/>} />
         <Route path="/categories" element={<CategoriesPage/>} />
+        <Route path="/rules" element={<RulesPage/>} />
       </Routes>
```
Comment (aniketsingh13) [inline src/App.js:14]: suggestion,you can make different file for router

**Review by aniketsingh13 — APPROVED** [review]
code is well structured and ui also look goods

File: src/pages/rulespage/RulesPage.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import "./RulesPage.css";
+import "../../data/rules"
+import { rules } from "../../data/rules";
```
Comment (govardhanreddyc79) [inline src/pages/rulespage/RulesPage.js:3]: Can use absolute path here

**Comment by govardhanreddyc79** [comment]
Structure looks good.
Clean and easily readable code

**Pull request merged**